packages/core/strapi/lib/services/metrics/admin-user-hash.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const crypto = require('crypto');
+
+const generateAdminUserHash = (ctx) => {
+  if (!ctx?.state?.user) {
```

Comment:
Why not use the requestContext here to get the user and avoid having to pass it as a param ?

Reply:
Changed it 👍 Although this would leave admin implementation of this still passing a param